FiveCalls/FiveCalls/Welcome.swift

```diff
@@ -9,112 +9,127 @@
-            Text(subMessage)
-                .font(.headline)
-                .textCase(.uppercase)
-                .foregroundStyle(.fivecallsDarkBlueText)
-                .opacity(subMessageOpacity)
-                .animation(.easeIn, value: subMessageOpacity)
-            Spacer()
-            Button(action: {
-                onContinue?()
-                dismiss()
-            }) {
-                Text(R.string.localizable.welcomePage2ButtonTitle())
-                    .fontWeight(.semibold)
-                    .foregroundColor(.white)
-                    .font(.system(size: 30))
-            }
-            .padding()
-            .frame(maxWidth: .infinity, minHeight: 73)
-            .background {
-                RoundedRectangle(cornerRadius: 6)
-                    .foregroundColor(Color.fivecallsDarkBlue)
+        ScrollView {
+            VStack {
+                Image(decorative: R.image.fivecallsLogotype)
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 292)
+                    .padding(.vertical, 24)
+
+                VStack(alignment: .leading, spacing: 30) {
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(R.string.localizable.welcomePage1Title())
+                                .fontWeight(.heavy)
+                                .multilineTextAlignment(.leading)
+                            Text(R.string.localizable.welcomePage1Message())
+                                .multilineTextAlignment(.leading)
+                        }
+                        .accessibilityElement(children: .combine)
+                    } icon: {
+                        Image(systemName: "phone.badge.waveform")
+                            .symbolRenderingMode(.palette)
+                            .foregroundStyle(.fivecallsRed, .fivecallsDarkBlue)
+                    }
+                    .labelStyle(CustomLabel(spacing: 20))
+
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(R.string.localizable.welcomePage2Title())
+                                .fontWeight(.heavy)
+                            Text(R.string.localizable.welcomePage2Message())
+                        }
+                        .accessibilityElement(children: .combine)
+                    } icon: {
+                        Image(systemName: "goforward.5")
+                            .foregroundStyle(.fivecallsDarkBlue)
+                    }
+                    .labelStyle(CustomLabel(spacing: 20))
+
+                    Label {
+                        VStack(alignment: .leading) {
+                            Text(subMessage)
+                                .fontWeight(.heavy)
+                                .opacity(subMessageOpacity)
+                        }
+                        .accessibilityElement(children: .combine)
```

Comment:
I haven't used any accessibility components in SwiftUI yet so I might be misunderstanding here, but in reading the docs for this it sounds like this would be used to combine multiple views into a single element that would used when iterating through accessible items on the page. What does this do if there's only one view within the VStack?

Reply:
Yep, we can leave this off.